includes/Experiments.php

```diff
@@ -378,6 +378,18 @@ public function get_experiments(): array {
 				'description' => __( 'Enable adding captions to background audio', 'web-stories' ),
 				'group'       => 'editor',
 			],
+
+			/**
+			 * Author: @barklund
+			 * Issue: #10112
+			 * Creation date: 2022-01-27
+			 */
+			[
+				'name'        => 'newDesignPanel',
+				'label'       => __( 'New Design Panel', 'web-stories' ),
+				'description' => __( 'Enable the new design panel integrated in the left-side menu including floating design bar', 'web-stories' ),
```

Comment:
Maybe let's have a flag just for the Floating menu? The main reason for adding this behind a feature flag and merging together with the right side panel change was that UX thought it might not fit otherwise. But if it fits, we could merge it earlier.

The main goal is not to get rid of the right panel before the floating menu. Having floating menu alone is ok though!

Reply:
Sure, done in 626c3738c.